src/SeedTokenAPIClientGeneral.js

```diff
@@ -0,0 +1,144 @@
+const axios = require('axios');
+const { Contract, utils, Wallet } = require('ethers');
+const SeedTokenAPIProviderAbstract = require('./SeedTokenAPIClientAbstract');
+const tokenAbi = require('./abis/token');
+const componentRepositoryAbi = require('./abis/componentRepository');
+const userRepositoryAbi = require('./abis/userRepository');
+
+const blockchainExplorerApiUrl = process.env.BLOCKCHAIN_EXPLORER_URL;
+const defaultOptions = {
+  baseURL: blockchainExplorerApiUrl,
+};
+
+/**
+ * @summary Handles all general blockchain calls
+ */
+class SeedTokenAPIClientGeneral extends SeedTokenAPIProviderAbstract {
```

Comment:
I think that we should add `checkAddress` that checks the provided string whether it is a valid ethereum address, as it was present in the previous version of the interfaces